components/shortcut/shortcut.app.js

```diff
@@ -0,0 +1,220 @@
+     * [CreateStoryCommentParams](https://shortcut.com/api/rest/v3#CreateStoryCommentParams)
+     * structure.
+     * @params {Date} completedAtOverride - A manual override for the time/date the Story was
+     * completed.
+     * @params {Date} createdAt - The time/date the Story was created.
+     * @params {Date} dueDate - The due date of the story.
+     * @params {string} description - The description of the story.
+     * @params {integer} epicId - The unique identifier of the epic the story belongs to.
+     * @params {integer} estimate - The numeric point estimate of the story. Can be null, which
+     * means unestimated.
+     * @params {string} externalId - This field can be set to another unique ID. In the case that
+     *  the Story has been imported from another tool, the ID in the other tool can be indicated
+     * here.
+     * @params {array} externalLinks - An array of External Links associated with this story.
+     * @params {array} fileIds - An array of IDs of files attached to the story.
+     * @params {array} followerIds - An array of UUIDs of the followers of this story.
+     * @params {string} groupId - The id of the group to associate with this story.
+     * @params {integer} iterationId - The ID of the iteration the story belongs to.
+     * @params {array} labels - An array of labels attached to the story. Each label must have the
+     * [CreateLabelParams](https://shortcut.com/api/rest/v3#CreateLabelParams) structure.
+     * @params {array} linkedFileIds - An array of integers with the IDs of linked files attached
+     * to the story.
+     * @params {string} name - The name of the story.
+     * @params {array} ownerIds - An array of UUIDs of the owners of this story.
+     * @params {integer} projectId - The ID of the project the story belongs to.
+     * @params {integer} requestedById - The ID of the member that requested the story.
+     * @params {Date} startedAtOverride - Manual override for the time/date the Story was started.
+     * @params {array} storyLinks - An array of story links attached to the story. Each story link
+     * must have the
+     * [CreateStoryLinkParams](https://shortcut.com/api/rest/v3#CreateStoryLinkParams) structure.
+     * @params {string} storyType - The type of story (feature, bug, chore).
+     * @params {array} tasks - An array of tasks connected to the story. Each task must have the
+     * [CreateTaskParams](https://shortcut.com/api/rest/v3#CreateTaskParams) structure.
+     * @params {string} updatedAt - The time/date the story was updated.
+     * @params {integer} workflowStateId - The ID of the workflow state the story will be in.
+     * @returns {story: object } An object with the created story, as per input provided, default
+     * values. See the full schema at
+     * [Create Story Responses](https://shortcut.com/api/rest/v3#Responses-79271).
+     */
+    async createStory(data) {
+      return await this.api().createStory(data);
```

Comment:
I believe if their are async the use of await is required.
Can you please make sure before I commit this suggestion.

Reply:
I think you don't even use this function right? because you are calling it using `this.api()` from `this.callWithRetry()`. 
The same with functions like `this.listEpics()`, `this.listFiles()`, `this.listLinkedFiles()`, `this.listMembers()`, `this.listProjects()` and `this.listWorkflows()` they are not being used

Reply:
You are right, good catch. This legacy code from that review comment when using callWithRetry was suggested.
We lost the documentation on api calls for this methos. I removed them but if we change opinion we can rollback.